report_thesis/src/sections/proposed_approach/model_selection.tex

```diff
@@ -0,0 +1,113 @@
+Similarly, \citet{yangConvolutionalNeuralNetwork2022} showed that \gls{cnn} outperformed methods such as \gls{logreg}, \gls{svm}, and linear discriminant analysis in correctly classifying twelve different types of rocks based on \gls{libs} data.
+While this example for \gls{cnn} involves a classification task, \gls{cnn} can be adapted for regression by changing the loss function and output layer.
+Based on these factors, we decided to include \gls{ann} and \gls{cnn} in our experiments to further increase the diversity of our model selection.
+
+To further bolster our selection pool, we included models from the same family as those that showed promise in the literature.
+
+\gls{xgboost} was included as an option based on its promising accuracy in various settings.
+For example, \citet{xgboost_in_biomedicie} showed that \gls{xgboost} outperformed models such as \gls{rf} and \gls{svm} in predicting biological activity based on quantitative description of the compound's molecular structure. 
+Another example is \citet{xgboost_in_heart_disease}, who used \gls{xgboost} to predict heart disease and found that \gls{xgboost} outperformed \gls{rf} and \gls{etr} in correctly classifying patients with heart disease.
+Due to these factors and the limited study of \gls{xgboost} in the context of \gls{libs} data, we decided to include it in our experiments.
+
+Following the same logic, we included \gls{ngboost}. 
+\gls{ngboost} is a recent model introduced by \citet{duan_ngboost_2020} that, according to their work, improves upon \gls{gbr} by using a more sophisticated loss function and a more advanced gradient boosting algorithm.
+Limited research has been conducted using this algorithm in the context of \gls{libs} data. 
+However, \citet{ngboost_landslide} showed that \gls{ngboost} outperformed \gls{xgboost} and \gls{rf} in correctly determining landslide-prone fields, with an AUC of 0.898 compared to 0.871 and 0.863 for \gls{xgboost} and \gls{rf}, respectively.
+
+Finally, \gls{ridge}, \gls{enet}, \gls{etr}, and \gls{lasso} were included in various studies and showed promising results, even if they were not the top performers in their respective studies.
+Therefore, we chose to include these in our experiments to further diversify our model selection. 
+
+Table~\ref{tab:preprocessing-models} summarizes the preprocessing techniques and models selected for our experimentation.
+
+\begin{table}[ht]
+\centering
+\begin{tabularx}{\columnwidth}{>{\raggedright\arraybackslash}X}
+\toprule
+\textbf{Normalization / Scaling:} \\
+\midrule
+Z-Score Normalization \\
+Min-Max Normalization \\
+Max Absolute Scaling \\
+Robust Scaling \\
+Norm 3 \\
+\midrule
+\textbf{Transformation Methods:} \\
+\midrule
+Power Transformation \\
+Quantile Transformation \\
+\midrule
+\textbf{Dimensionality Reduction Methods:} \\
+\midrule
+PCA \\
+Kernel PCA \\
```

Comment:
```suggestion
\gls{kernel-pca} \\
```

Reply:
This *still* renders it as an acronym, while all the others aren't. 
My point is that all the other listed methods and techniques are not listed using their acronyms, only PCA is. This is inconsistent.